.github/workflows/ci.yml

```diff
@@ -144,46 +144,46 @@ jobs:
     runs-on: ubuntu-latest
     if: ${{ always() && contains(join(needs.*.result, ','), 'success') }}
     needs: [Linting, Typing, Pytests]
```

Comment:
Unrelated to this PR but I've noticed the notebooks still running even if a previous step fails.  I was reading more on this and I think we only need the `needs` property to force the success dependency, so we might want to just drop the `if` statement.